.github/workflows/behave.yml

```diff
@@ -54,12 +57,19 @@ jobs:
       - name: Run Application
         run: make run
 
+      - name: Check Docker state (pre-test)
+        run: docker ps
+
       - name: Run pytest + behave with Coverage
         env:
           POSTGRES_USER: scram
           POSTGRES_DB: test_scram
         run: make coverage.xml
 
+      - name: Dump docker logs on failure
+        if: failure()
```

Comment:
Probably fine, but I could see where this is more of a debug and we don't want to include these debug things? I can go either way.

Reply:
I don't mind having the debug steps as part of the GitHub Action, as long as it's not part of the Behave testing itself. Longer term, I'd like to figure out how to surface warnings from within Docker as well, but right now, viewing the logs is the only way to see. It did surface a legit issue that our tests right now don't seem to notice when the Django container never starts, which is troubling.